### PR TITLE
fix: Switch to using D2L dialog/producer instead of webcomponent

### PR DESCRIPTION
**Pull request by AdmiralCoco**
- Attempt to use native dialog to fix iOS cut off issue

There should be no UI changes (visually), though I am unable to test on iOS as I can't access my local machine - so I am planning to test this again once merged

### CODE REVIEW

File: components/d2l-activity-editor/d2l-activity-content-editor/content-file/d2l-activity-content-media-file-detail.js [hunk 1]
```diff
@@ -132,11 +129,37 @@ class ContentMediaFileDetail extends SkeletonMixin(ErrorHandlingMixin(LocalizeAc
...
+							topWindow = handle.win.top;
+							onResize();
+							topWindow.addEventListener('resize', onResize);
+						}, 500);
+					}
+				}
+			);
+
+			dialogResult.AddReleaseListener(() => {
+				this._refetchCaptions(mediaFileEntity);
```
Comment (AdmiralCoco) [inline components/d2l-activity-editor/d2l-activity-content-editor/content-file/d2l-activity-content-media-file-detail.js:160]: This should keep the behaviour of refetching the captions on close

**Review by MichaelKovalchuk — APPROVED** [review]

**Comment by MichaelKovalchuk** [comment]
> * Attempt to use native dialog to fix iOS cut off issue
> 
> There should be no UI changes (visually), though I am unable to test on iOS as I can't access my local machine - so I am planning to test this again once merged

iOS Dialog cutoff issue
FACE_iOS_advanced_editing_dialog_cut_off

**Review by coffeexcode — APPROVED** [review]
LGTM

**Comment by MichaelKovalchuk** [comment]
@coffeexcode The issue we ran into with the non-legacy dialogs seems to be FACE specific. Only reproducible in iOS. Safari on MAC works fine

**Pull request merged**